coordinator/clientapi/clientapi_test.go

```diff
@@ -120,6 +121,14 @@ func TestGetCertQuote(t *testing.T) {
 			},
 			wantErr: true,
 		},
+		"get quote with nonce": {
+			store: prepareDefaultStore(),
+			core: &fakeCore{
+				state: state.AcceptingMarbles,
+				quote: []byte("quote"),
+			},
+			nonce: []byte("nonce"),
```

Comment:
for a meaningful test, the fakeCore should do something with the nonce and we should have wantQuote